config/purposes/library_dilution_plate.wip.yml

```diff
@@ -0,0 +1,5 @@
+Library Stock Dil Plate:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate: false
+  :creator_class: LabwareCreators::StampedPlate
```

Comment:
Note: Direct stamp to preserve sample identities and positions across plates. The **parent** plate Library Stock Plate is an input plate. Sequencescape has a comment about it at config/default_records/plate_purposes/005_limber_purposes.yml : "These used to be imported as Stock Plates and then 'fake cherrypicked' onto 'LB Lib PCR XP'".